git/gittest/repo.go

```diff
@@ -38,6 +38,10 @@ func Repo(t *testing.T) (git.Repo, func()) {
 		cleanup()
 		t.Fatal(err)
 	}
+	if err = execCommand("git", "-C", filesDir, "tag", "flux-test"); err != nil {
```

Comment:
Ah, not quite true.
I think SyncStatus requires a flux tag. Since it does a revision search from `flux-tag..$HASH`.

So I'll need to add the tag for this one.